src/core/etl/src/Flow/ETL/DSL/From.php

```diff
@@ -63,14 +64,13 @@ final public static function data_frame(DataFrame $data_frame) : Extractor
 
     /**
      * @param Memory $memory
-     * @param int<1, max> $chunkSize
-     * @param string $rowEntryName
+     * @param int<1, max> $chunk_size
      *
      * @return Extractor
      */
-    final public static function memory(Memory $memory, int $chunkSize = 100, string $rowEntryName = 'row') : Extractor
+    final public static function memory(Memory $memory, int $chunk_size = 100, EntryFactory $entry_factory = new NativeEntryFactory()) : Extractor
```

Comment:
Changing `camelCase` to `snake_case` should also be considered as a BC break.

I think we need to agree globally on this cause some code uses one thing others different one.

Reply:
I did not document it anywhere, but the whole DSL should use a snake case for all function names/arguments.
Those are simply leftovers that I missed.